staging/traefik/values.yaml

```diff
@@ -136,7 +136,7 @@ ssl:
   # tlsMinVersion: VersionTLS12
   # https://docs.traefik.io/configuration/entrypoints/#strict-sni-checking
   # sniStrict: false
-  customCert: false
```

Comment:
what about doing something like https://github.com/mesosphere/charts/pull/369/files#diff-84491f379c2eed7f94384116a9e49e45R74 with a boolean and a secret name ?

Reply:
Sorry, didn't understand your comment. This variable was introduced by us and this patch is taking it out

Reply:
I meant, why don't you respect the same structure and have a complex object with a boolean and a secretName as done in https://github.com/mesosphere/charts/pull/369/files#diff-84491f379c2eed7f94384116a9e49e45R74

Reply:
What is the benefit of that structure?